cli/integration_tests/monorepo_one_script_error/run.t

```diff
@@ -33,7 +33,7 @@ Make sure it isn't cached
   \xe2\x80\xa2 Packages in scope: my-app (esc)
   \xe2\x80\xa2 Running error in 1 packages (esc)
   \xe2\x80\xa2 Remote caching disabled (esc)
-  my-app:error: cache miss, executing aed5e5ad070dbe8e
+  my-app:error: cache miss, executing [0-9a-f]+ (re)
```

Comment:
This is clever, but I think we actually care to fail tests when the hash changes. Could you make it the real thing?